packages/taquito-local-forging/src/Proto12-Ithaca/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { kindMapping } from '../constants';
```

Comment:
not really a comment on the code, but maybe it would be better to name the folder just `ithaca` or `proto12-ithaca` in lower kebab-case for consistency?